Yoast/wordpress-seo
# Add Norwegian passive voice assessment

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the passive voice assessment for Norwegian.
* [Yoast SEO Free] Adds the assessment for detecting passive voice in Norwegian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Norwegian.
* Test this functionality in Yoast SEO Free.

* Create a new empty post and add these sentences to your post:
`Den er stengt og ble låst hele dagen.`
`Den ble lovet, det vil bli gjort.`
`Eksemplet ble erstattet.`
`Det vil bli gjort`
`Huset er bygget med murstein.`
`Nyhetene ble kunngjort i tide.`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add these sentences and check that that they do not get highlighted:
`Cameraet er her har vært vitne til oss.`
`Dette er en veldig god idé.`

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-713
